lib/torrent.js

```diff
@@ -323,6 +323,13 @@ class Torrent extends EventEmitter {
       })
     }
 
+    // add BEP09 peer-address
+    // TODO: It should only be included if the client can discover its public IP address and determine its reachability.
+    if (this.peerAddress) {
```

Comment:
This needs to be `peerAddresses` since the API changed in `magnet-uri`.

Reply:
Fixed! 👍